src/screens/StarterPack/Wizard/StepProfiles.tsx

```diff
@@ -45,8 +45,9 @@ export function StepProfiles({
     const seen = new Set<string>()
     return topPages.pages
       .flatMap(p => p.actors)
+      .filter(p => !p.associated?.labeler)
       .filter(p => {
-        if (!p.associated?.labeler && !seen.has(p.did)) {
+        if (!seen.has(p.did)) {
```

Comment:
i broke this out in a separate filter 'cause i don't really know what the `labeler` is and didn't want to break anything